mongo/src/main/scala/io/sphere/mongo/format/DefaultMongoFormats.scala

```diff
@@ -71,4 +71,39 @@ trait DefaultMongoFormats {
       }
     }
   }
+
+  implicit def listFormat[A](implicit f: MongoFormat[A]): MongoFormat[List[A]] = new MongoFormat[List[A]] {
+    import scala.collection.JavaConverters._
+    override def toMongoValue(a: List[A]) = {
+      val m = new BasicDBList()
+      m.addAll(a.map(f.toMongoValue(_).asInstanceOf[AnyRef]).asJavaCollection)
+      m
+    }
+    override def fromMongoValue(any: Any): List[A] = {
+      any match {
+        case l: BasicDBList =>
+          val it = l.asInstanceOf[BasicDBList].iterator()
+          it.asScala.map(f.fromMongoValue).toList
```

Comment:
what about using `(breakOut)` to avoid the `toList`?

Reply:
Good point, but `map` of `Iterator` does not take any `CanBuildFrom`:
```
trait Iterator[+A] extends TraversableOnce[A] {
  self =>
    def map[B](f: A => B): Iterator[B]  = new AbstractIterator[B] {
    def hasNext = self.hasNext
    def next() = f(self.next())
  }
}
```